lib/active_merchant/billing/gateways/authorize_net.rb

```diff
@@ -544,6 +544,7 @@ def valid_track_data
       def add_check(xml, check)
         xml.payment do
           xml.bankAccount do
+            xml.accountType(check.account_type)
```

Comment:
Is there ever a situation where there wouldn't be an account_type for a check or is it always defined? (I'm not super familiar with this payment method)

Reply:
Looks like it is required https://github.com/activemerchant/active_merchant/blob/1b2fce718fe211865297805ebe634e5764a8d777/lib/active_merchant/billing/check.rb#L40